### ISSUE TITLE
Sad anime girl for 404

### ISSUE THREAD

**Issue by PalAditya** [asheeeshh/waifus-for-programmers#20]
This is insanely good 😄 
However, the 404 looks a but plain. How about we add a sad waifu to the 404 page? I can help contribute too if needed

**Comment by AeriaVelocity** [comment]
Seconded, this would fit in well

**Comment by asrvd** [comment]
> This is insanely good 😄 However, the 404 looks a but plain. How about we add a sad waifu to the 404 page? I can help contribute too if needed

i like the idea, would love to see you implement it, i'm also planning to make some improvements on the site so it would be a good headstart

**Comment by AeriaVelocity** [comment]
It wouldn't be too hard, just have a random girl show up with "404" or "Not found" somewhere in the image on the 404 page, right?

**Comment by asrvd** [comment]
> It wouldn't be too hard, just have a random girl show up with "404" or "Not found" somewhere in the image on the 404 page, right?

yes, something like that but i won't be free till next week so cant work on it myself for now

**Comment by AeriaVelocity** [comment]
> yes, something like that but i won't be free till next week so cant work on it myself for now

No problem, we can wait, or if anyone else wants to do it, that'd be good too

**Comment by PalAditya** [comment]
Cool, I should be able to send a PR soon.
